template/web/app/paymaster-bundler/_components/PaymasterBundlerDemo.tsx

```diff
@@ -0,0 +1,136 @@
+import clsx from 'clsx';
+import { useAccount } from 'wagmi';
+import { useWriteContracts } from 'wagmi/experimental';
+import isLocal from '../../../src/utils/isLocal';
+import { usePaymasterBundlerContract } from '../_contracts/usePaymasterBundlerContract';
+import { CallStatus } from './CallStatus';
+
+// Use the local API URL to target the Paymaster directly without a proxy
+// if running on localhost, otherwise use the Paymaster Proxy.
+const isLocalEnv = isLocal();
+const defaultUrl = isLocalEnv
+  ? `https://api.developer.coinbase.com/rpc/v1/base-sepolia/z7inYI-NRNAOF9kgaW4Suf-30N6DuMra`
+  : `${document.location.origin}/api/paymaster-proxy`;
+
+console.log('Default url: ', defaultUrl);
+
+export default function PaymasterBundlerDemo() {
+  const account = useAccount();
+  const { data: id, writeContracts } = useWriteContracts();
+
+  const contract = usePaymasterBundlerContract();
+
+  if (contract.status !== 'ready') {
+    console.error('Contract is not ready');
+    return null;
+  }
+
+  const handleMint = () => {
+    console.log('Default url: ', defaultUrl);
+    console.log('account: ', account);
```

Comment:
All this console.log are all over the place. If want to use them, we should have a more clean format.

Reply:
Yeah sorry these were more for me to help debug. Removed!